packages/flutter/lib/src/services/raw_keyboard_linux.dart

```diff
@@ -19,16 +19,16 @@ import 'raw_keyboard.dart';
 class RawKeyEventDataLinux extends RawKeyEventData {
   /// Creates a key event data structure specific for macOS.
   ///
-  /// The [toolkit], [scanCode], [codePoint], [keyCode], and [modifiers], arguments
-  /// must not be null.
+  /// The [toolkit], [scanCode], [charactersIgnoringModifiers], [keyCode], and
+  /// [modifiers], arguments must not be null.
   const RawKeyEventDataLinux({
     @required this.keyHelper,
+    this.charactersIgnoringModifiers = '',
     this.scanCode = 0,
-    this.codePoint = 0,
```

Comment:
Does this need a breaking change proposal, since it's related to engine/platform?
+cc @Hixie

Reply:
Keyboard mapping is still a WIP, the linux part hasn't been implemented by the engine (will be after this PR is merged). I don't think it'd be worth it to add a proposal for this since these apis are still subject to change quickly and desktop is still not a stable platform.

Reply:
We should still announce it on flutter-announce and label this as a break if it is a break.